hack/patch-csv.sh

```diff
@@ -98,7 +98,7 @@ if [ $# -ne 1 ]; then
 fi
 clusterserviceversion_file_path="$1"
 
-versions="$( ${YQ} '.versions[].name' "${VERSIONS_YAML_FILE}" )"
+versions="$( ${YQ} '.versions[].name' "pkg/istioversions/${VERSIONS_YAML_FILE}" )"
```

Comment:
Isn't there an environment variable for the version file dir? Why aren't we using it here?